README.md

```diff
@@ -30,8 +30,8 @@ To load the dumped data back into a local Postgres:
 ```sh
 $ dropdb cfgov
 $ createdb cfgov
-$ createuser cfgov
-$ gunzip < postgres.sql.gz | psql cfgov
+$ createuser cfpb
+$ gunzip < postgres.sql.gz | psql postgres://cfpb@localhost/cfgov
```

Comment:
@richaagarwal sorry, I missed this comment before merging in. I didn't change the name of the database to `cfpb`, only the username. The database name used here is still `cfgov`.

That database name is only used internally to this project so doesn't have any impact on databases that use the converted dump.

Reply:
Oh no worries, it was my fault -- I had approved and then later left this comment.  

Hmm, are you sure you didn't change the database name? https://github.com/cfpb/cfgov-mysql-to-postgres/pull/5/files#diff-4e5e90c6228fd48698d074241c2ba760R10  Or am I missing something?

Reply:
Oh, I see what I did. I changed the database names used internally by the MySQL and Postgres DB containers but didn't change the README example of how you might load that dump.

Technically the README is still valid. You can load the PG dump into a database with any name. It will be loaded with a schema of `cfpb`, but the database name can be anything (consider for example our various database names).

Personally I might prefer to keep a convention of our local development databases being named something like `cfgov` even if our database username is `cfpb`. That feels a little clearer to me especially if you have other CFPB-related databases in your local Postgres. Having `cfpb` everywhere for consistency is simpler, although it obscures a bit exactly what are real requirements vs. preferences. I'm happy to change everywhere to be consistent if you think it'll be clearer.

What if I change the README to something like:

---

To load the dumped data back into a local Postgres database named `cfgov`:

```
instructions
```

Postgres dumps contain tables and data in the `cfpb` schema. For this reason you may want to access the loaded data with a user named `cfpb` as well. Otherwise you'll have to modify the Postgres [schema search path](https://www.postgresql.org/docs/10/static/ddl-schemas.html) to include the `cfpb` schema.